### ISSUE TITLE
this repository is outdated

### ISSUE THREAD

**Issue by conradsnicta** [Zefiros-Software/Armadillo#1]
This repo mirrors the Armadillo SVN repo, which is no longer used.

The new official Armadillo source code repository is:
https://github.com/conradsnicta/armadillo-code

**Comment by PaulVisscher** [comment]
Yeah I have seen your effort in migrating to GitHub! I was planning to shut this repo down somewhere in the next 2 weeks, and migrate to the official repo. I have a minor suggestion for the current repo, and since there is no issue tracker enabled I will drop it here. Could you use git tags to mark commits as their respective versions? This makes using a git workflow somewhat easier. See [https://github.com/boostorg/boost](https://github.com/boostorg/boost) for example how they handle their releases. Keep up the awesome work!

**Comment by conradsnicta** [comment]
I'd like to keep the bureaucracy to a minimum, so I don't use git tags. If you want a specific release, please obtain it from the official download page:
http://arma.sourceforge.net/download.html

The git repo has currently two branches: `devel` and `7.600.x`.  The `devel` branch is for continuous development (potentially containing unstable code), while the `7.600.x` branch is only for bug fixes.  It will continue to receive bug fixes until version 7.700 is released (and a new branch created).
